docs/advanced.md

```diff
@@ -138,6 +138,19 @@ There's a nice example of this being done in CircleCI [at Synthetix, here][31].
 
 :bulb: **Pro Tip**: Codecov CI will automatically combine coverage reports sent to them as a batch - if you're using that service you don't need to do this yourself.
 
+## Coverage threshold checks
+
+Istanbul has a command line utility which can be used to set thresholds for different coverage categories and throw an error if your coverage drops below them. (Istanbul is a solidity-coverage dependency so you shouldn't need to install it separately.)
```

Comment:
Link to sc-istanbul's bin command:

https://github.com/sc-forks/istanbul/blob/795f7a4f1bfd52378f475303da78e18bd35da0f9/package.json#L99-L101